src/v0/sources/shopify/util.js

```diff
@@ -140,42 +141,31 @@ const setAnonymousIdorUserIdFromDb = async (message, metricMetadata) => {
     case SHOPIFY_TRACK_MAP.orders_partially_fullfilled:
     case RUDDER_ECOM_MAP.orders_create:
     case RUDDER_ECOM_MAP.orders_updated:
-
-      if (!isDefinedAndNotNull(message.properties?.cart_token)) {
-        /**
-         * This case will rise when we will be using Shopify Admin Dashboard to create, update, delete orders etc.
-         * Since it is done by shopify-admin we will set "userId" to be "shopify-admin"
-         */
-        if (!message.userId) {
-          message.setProperty('userId', 'shopify-admin');
-        }
-        return;
-      }
-      cartToken = message.properties.cart_token;
+      cartToken = properties?.cart_token;
       break;
     /*
      * we dont have cart_token for carts_create and update events but have id and token field
      * which later on for orders become cart_token so we are fethcing cartToken from id
      */
     case SHOPIFY_TRACK_MAP.carts_create:
     case SHOPIFY_TRACK_MAP.carts_update:
-      cartToken = message.properties?.id || message.properties?.token;
+      cartToken = properties?.id || properties?.token;
       break;
     // https://help.shopify.com/en/manual/orders/edit-orders -> order can be edited through shopify-admin only
     // https://help.shopify.com/en/manual/orders/fulfillment/setting-up-fulfillment -> fullfillments wont include cartToken neither in manual or automatiic
     case SHOPIFY_TRACK_MAP.orders_edited:
     case SHOPIFY_TRACK_MAP.orders_delete:
     case SHOPIFY_TRACK_MAP.fulfillments_create:
     case SHOPIFY_TRACK_MAP.fulfillments_update:
```

Comment:
If there is not any computation to do we can remove these cases.

Reply:
Done